source/Frontend/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(QTFrontend VERSION 1.0.0 LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
use standard 20

Reply:
I tried with 20, but then qt building didn't go through. I don't know why, but it works now.